imagetool/src/main/resources/docker-files/install-middleware.mustache

```diff
@@ -35,9 +35,13 @@ USER {{userid}}
 RUN echo "INSTALLING MIDDLEWARE" \
 {{#installPackages}}
     && echo "INSTALLING {{type}}" \
-    && {{#isZip}}unzip -q {{{tempDir}}}/{{installerFilename}} "*.jar" -d {{{tempDir}}} &&{{/isZip}} \
+    && {{^isBin}}{{#isZip}}unzip -q {{{tempDir}}}/{{installerFilename}} "*.jar" -d {{{tempDir}}} &&{{/isZip}} \
     {{{java_home}}}/bin/java -Xmx1024m -jar {{{tempDir}}}/{{jarName}} -silent ORACLE_HOME={{{oracle_home}}} \
-    -responseFile {{{tempDir}}}/{{responseFile.name}} -invPtrLoc {{inv_loc}}/oraInst.loc -ignoreSysPrereqs -force -novalidation \
+    -responseFile {{{tempDir}}}/{{responseFile.name}} -invPtrLoc {{inv_loc}}/oraInst.loc -ignoreSysPrereqs -force -novalidation {{/isBin}} \
+    {{#isBin}}{{#isZip}}unzip -q {{{tempDir}}}/{{installerFilename}} "*.bin" -d {{{tempDir}}} &&{{/isZip}} chmod +x {{{tempDir}}}/{{jarName}} && \
+    {{{tempDir}}}/{{jarName}} -printtime -printmemory -printdiskusage -force -ignoreSysPrereqs -silent \
```

Comment:
Why are the options "-printtime -printmemory -printdiskusage" needed?

Reply:
OID team had suggested these optional options. Do you want to remove them?

Reply:
Checked with OID team and removed the above optional parameters.
https://github.com/brahankv/weblogic-image-tool/commit/d2d741bbc183844997efa53edb0ff7b05fa5b997